GoogleChromeLabs/ps-analysis-tool
# Feature: Add collapsible sidebar

## Description
- This PR adds logic and UI to show a collapsible sidebar.


## Relevant Technical Choices
- A new type, `CollapsedSidebarItems`, needs to be submitted to the provided props to allow collapsible sidebar logic to run.


## Testing Instructions
- Open PSAT panel and find double arrow beside Privacy sandbox item.
- Click on the arrow, the sidebar should collapse and screen will be expanded.
- Now clicking on the double arrow should expand the sidebar.


## Additional Information:



## Screenshot/Screencast



https://github.com/user-attachments/assets/3e445e64-fed3-403a-be9b-38079eabb719





---

## Checklist






Fixes #
